WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/CategorySectionTableViewCell.swift

```diff
@@ -51,6 +52,7 @@ class CategorySectionTableViewCell: UITableViewCell {
     }
 
     var isGhostCell: Bool = false
+    var ghostThumbnailSize: CGSize = defaultThumbnailSize
```

Comment:
This is clunky because of the way ghost thumbnails work - they aren't backed by any data source, so we can't find the height by querying the section (it will be nil).